_includes/ballot-nav.html

```diff
@@ -4,17 +4,20 @@
 {% assign ballots = site.ballots | where: "locality", locality.locality_id | reverse %}
 
 <nav class="ballot-nav">
-  {% if referendums and referendums != empty %}
-    <section class="l-section">
-      <div class="l-section__content">
-        <h3>{% include referendum_measure_heading.html referendum_measure_display=locality.referendum_measure_display %}</h3>
-        {% for ballot_item in referendums %}
-          {% assign referendum = site.referendums | where: "path", ballot_item | first %}
-          <p><a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a></p>
-        {% endfor %}
-      </div>
-    </section>
-  {% endif %}
+  <h2>Elective Offices</h2>
+  {% comment %}
+    {% if referendums and referendums != empty %}
+      <section class="l-section">
+        <div class="l-section__content">
+          <h3>{% include referendum_measure_heading.html referendum_measure_display=locality.referendum_measure_display %}</h3>
+          {% for ballot_item in referendums %}
+            {% assign referendum = site.referendums | where: "path", ballot_item | first %}
+            <p><a href="{{ referendum.url | prepend: site.baseurl }}">{% if referendum.number %}{{ referendum.number }} {% endif %}{{ referendum.title | escape}}</a></p>
+          {% endfor %}
+        </div>
+      </section>
+    {% endif %}
+  {% endcomment %}
```

Comment:
You can just delete this, git will allow us to bring it back.

Reply:
Will do!

Reply:
So just to be clear, you want robots.txt line 2 to say "Disallow: /site/"?